src/app/support/page.tsx

```diff
@@ -7,8 +7,8 @@ import EstTimelineSection from "./estTimeline";
 
 export default function SupportPage() {
   return (
-    <div className="SupportPage flex max-w-[1600px] flex-col self-center p-14 pb-0">
-      <div className="DetailsAndDonation flex justify-between gap-10">
+    <div className="SupportPage flex max-w-[100%] flex-col self-center py-8 pb-0 lg:max-w-[1600px] lg:p-14">
+      <div className="DetailsAndDonation flex flex-col justify-between gap-10 lg:flex-row">
         <StatusTimelineGoalsSection />
         <DonationBox />
       </div>
```

Comment:
The crowdfunding rewards section doesn't look great at smaller sized (see 320px). The animation is weird because the infoboxes are too big.